app/setup_handlers.go

```diff
@@ -34,3 +37,12 @@ func SetupHandlers(app *App) {
 		app.SetStoreLoader(types.UpgradeStoreLoader(upgradeInfo.Height, &storeUpgrades))
 	}
 }
+
+type VersionMigrator struct {
+	v module.VersionMap
+}
+
+func (v VersionMigrator) TriggerMigration(moduleName string) module.VersionMap {
```

Comment:
Why is it needed if we increase `ConsensusVersion`?

Reply:
We have always been using this same process 
We set the entire map to latest version 
	```
	for m, mb := range app.mm.Modules {
			vm[m] = mb.ConsensusVersion()
		}
```